sagemaker-training-compiler/huggingface/pytorch_tune_batch_size/batch_tune.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<h3>Wait for the Training Jobs to Complete</h3><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>